3rd/TokenBucket/TokenBucket.h

```diff
@@ -48,11 +48,7 @@ class TokenBucket {
         return *this;
     }
 
-    bool consume(const uint64_t tokens) {
-        const uint64_t now =
-                std::chrono::duration_cast<std::chrono::microseconds>(
-                        std::chrono::steady_clock::now().time_since_epoch())
-                        .count();
+    bool consume(const uint64_t tokens, const uint64_t now) {
```

Comment:
Probably best to document what `now` means. We can point to the libuv documentation:

http://docs.libuv.org/en/v1.x/loop.html#c.uv_now